graphql/aggregator.js

```diff
@@ -25,6 +29,57 @@ export default class Aggregator {
     return this;
   };
 
+  withNearText = (nearTextObj) => {
+    if (this.includesNearMediaFilter) {
+      throw new Error(
+        "cannot use multiple near<Media> filters in a single query"
+      )
+    }
+
+    try {
+      this.nearTextString = new NearText(nearTextObj).toString();
+    } catch (e) {
+      this.errors = [...this.errors, e];
+    }
+
+    this.includesNearMediaFilter = true
```

Comment:
imo this statement should be put between try/catch

same applies for remaining usecases

Reply:
ok 👍